index.js

```diff
@@ -20,3 +20,30 @@ module.exports = fn => {
 
 	return ret;
 };
+
+const log = (fn, promise) => {
+	console.log(`Promise from ${fn.displayName} resolved in ${promise.time} ms`);
```

Comment:
`fn` is the wrapped function passed in here, so I think it should be `displayName` as it is set on line `19`.

> fallback to [anonymous] when it's an empty string.

👍